src/WattleScript.Interpreter/DataTypes/WattleMemberType.cs

```diff
@@ -0,0 +1,13 @@
+namespace WattleScript.Interpreter
+{
+    public enum WattleMemberType
```

Comment:
Shouldn't be public if there is no exposing API yet. This would also be better as a `[Flags]` enum

Reply:
Fixed in 8462beb